### DISCUSSION TITLE
It doesn't seem to generate stimulus controllers? Is there a way I can enable or is it by design?

### DISCUSSION THREAD

**Discussion by SyedMSawaid** [palkan/view_component-contrib#45] [Q&A] [answered]
Same as the title

**Comment by palkan** [comment, marked as answer]
> Is there a way I can enable or is it by design?

We provide a minimal generator; you can upgrade it to generate Stimulus controllers as well.

We dropped the whole part on JS/CSS from the generator since it became too costly to support all the possible assets pipelines/managers/builders/no-builders. (Yeah, the only thing that was easier with Webpacker is that it was the only option and we could only take care of it).

**Reply by SyedMSawaid** [reply]
Alright man. Thanks for this wonderful gem. <3
